lib/Pages/home_screen.dart

```diff
@@ -200,14 +201,21 @@ class _MenuState extends State<Menu> {
             NavDrawerListTile(
                 icon: Icons.exit_to_app,
                 onTap: () async {
-                  controller.toggle();
-                  SharedPreferences prefs =
-                      await SharedPreferences.getInstance();
-                  prefs.setString('floodToken', '');
-                  Provider.of<UserDetailProvider>(context, listen: false)
-                      .setToken('');
-                  Navigator.of(context).pushNamedAndRemoveUntil(
-                      Routes.loginScreenRoute, (Route<dynamic> route) => false);
+                 showDialog(
+                      context: context,
+                      builder: (context) =>
+                          LogOutAlert(logoutOnClick: () async {
+                            controller.toggle();
+                            SharedPreferences prefs =
+                                await SharedPreferences.getInstance();
+                            prefs.setString('floodToken', '');
+                            Provider.of<UserDetailProvider>(context,
+                                    listen: false)
+                                .setToken('');
+                            Navigator.of(context).pushNamedAndRemoveUntil(
+                                Routes.loginScreenRoute,
+                                (Route<dynamic> route) => false);
+                          }));
```

Comment:
This line is causing formatting issues with the workflow. Please make sure you follow the flutter linting convention and add "," after each ")" to ensure correct formatting.

Reply:
Noted sir .

Reply:
There still seem to be some linting issues. You can run ```flutter format -n --set-exit-if-changed .``` locally to deug it.